translations/hu.json

```diff
@@ -0,0 +1,2984 @@
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "removedRequirement": [
+        "A **{requirement}** követelmény el lett távolítva a **{guild}** nevű guild-ról!"
+      ]
+    }
+  },
+  "config guilds edit-requirement": {
+    "name": "követelmény-módosítás",
+    "description": "Módosítja egy Hypixel guild követelményét.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A guild, ahol módosítod a követelményt."
+      },
+      {
+        "name": "követelmény",
+        "description": "A módosítandó követelmény.",
+        "choices": [
+          "Hypixel szint",
+          "Képesség átlag",
+          "Slayer tapasztalat",
+          "Minion helyek",
+          "Catacombs szint",
+          "Fairy Souls",
+          "Súlyozás"
+        ]
+      },
+      {
+        "name": "érték",
+        "description": "A követelmény új értéke."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds guild-hozzáadása` paranccsal."
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "requirementNotFound": [
+        "A **{requirement}** követelmény nem létezik a **{guild}** nevű guild-ban."
+      ],
+      "updatedRequirement": [
+        "A **{requirement}** követelmény **{value:,}**-re lett állítva a **{guild}** nevű guild-ban!"
+      ]
+    }
+  },
+  "config guilds add-rank-requirement": {
+    "name": "rang-követelmény-hozzáadás",
+    "description": "Hozzáad egy követelményt egy Hypixel guild rangjához.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A guild amelyiknél a rang van."
+      },
+      {
+        "name": "rank",
+        "description": "A rang, amelyikhez a követelményt adod."
+      },
+      {
+        "name": "requirement",
+        "description": "A hozzáadandó követelmény.",
+        "choices": [
+          "Hypixel szint",
+          "Képesség átlag",
+          "Slayer tapasztalat",
+          "Minion helyek",
+          "Catacombs szint",
+          "Fairy Souls",
+          "Súlyozás",
+          "API beállítások bekapcsolva"
+        ]
+      },
+      {
+        "name": "érték",
+        "description": "A követelmény értéke. Írj be akármilyen számot a 'API beállítások bekapcsolva' követelményhez."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds add-guild` paranccsal."
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "rankNotFound": [
+        "A **{rank}** rang nem létezik a **{guild}** nevű guild-ban."
+      ],
+      "rankNotAdded": [
+        "A **{guild}** nevű guild **{rank}** rangja nincs regisztrálva a bottal.\n",
+        "Regisztráld a `/config guilds add-rank` paranccsal."
```

Comment:
The whole command name should be translated, including top level
Should also be in backticks for `this effect`